### PR TITLE
Fix titles case and add missing punctuation marks

### PR DESCRIPTION
**Pull request by Rowayda-Khayri**
This PR fixes titles case and adds missing punctuation marks in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Grafana allows you to query, visualize, alert on and understand your metrics no
 - **Alerting:** Visually define alert rules for your most important metrics. Grafana will continuously evaluate and send notifications to systems like Slack, PagerDuty, VictorOps, OpsGenie.
 - **Mixed Data Sources:** Mix different data sources in the same graph! You can specify a data source on a per-query basis. This works for even custom datasources.
 
-## Get started
+## Get Started
```
Comment (torkelo) [inline README.md:18]: we do not use title case

Comment (achatterjee-grafana) [reply README.md:18]: Correct, we use sentence case. 

Please make the following change:
"Mixed data sources" and "Get started".

File: README.md [hunk 2]
```diff
@@ -34,10 +34,10 @@ If you're interested in contributing to the Grafana project:
 - Learn how to set up your local environment, in our [Developer guide](/contribute/developer-guide.md).
 - Explore our [beginner-friendly issues](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22).
 
-## Get involved
+## Get Involved
```
Comment (achatterjee-grafana) [inline README.md:37]: Revert the change.

**Review by achatterjee-grafana — COMMENTED** [review]
Added copy-edit suggestions.

File: README.md (hunk 2 above)
Comment (Rowayda-Khayri) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (Rowayda-Khayri) [reply README.md]: Done

Comment (achatterjee-grafana) [reply README.md]: @Rowayda-Khayri was this fixed? Thanks!

**Review by achatterjee-grafana — COMMENTED** [review]
Added a comment.

File: README.md (hunk 1 above)
Comment (Rowayda-Khayri) [reply README.md]: > @Rowayda-Khayri was this fixed? Thanks!

"Get Started" was fixed. But "Mixed data sources" wasn't changed in my PR!

Comment (achatterjee-grafana) [reply README.md]: OK - would you want to fix it now or should we do a new PR? Let me know :)

**Review by achatterjee-grafana — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (Rowayda-Khayri) [reply README.md]: @achatterjee-grafana OK, I'll make a new PR for it
